### DISCUSSION TITLE
Specmatic Example Is failing when trying to test service.yaml

### DISCUSSION THREAD

**Discussion by Kpizzle** [specmatic/specmatic#718] [Ask For Help] [answered]
I'm currently looking to see if we could use specmatic in my QA team, and I'm following the example in the documentation to test the petstore. 

However when I try to run the provider side-contract test, nothing comes back. My terminal just runs the command, and reports no output. There is no error or success message like I'm expecting from the docs. 

My steps are as follows - 

- install specmatic with `npm install -g specmatic`
- confirm installation with `$ npx specmatic`
- created and saved the openAPI doc
- in my terminal, navigate to that folder
- once in the correct folder run the command `npx specmatic test service.yaml --testBaseURL=https://my-json-server.typicode.com/znsio/specmatic-documentation`
- I'm then expecting an output as per the docs, but there is nothing displayed in the terminal. 

I've installed specmatic via npm. Running on MacOS Ventura 13.5.1.

**Comment by harikrishnan83** [comment, marked as answer]
@Kpizzle Thanks for reaching out to us. This indeed seems to be an issue with the npm module and / or the  documentation. We are taking a look.

**Reply by Kpizzle** [reply]
Awesome. Glad to have helped. Hopefully it's a quick fix and I can get going with trying it out.

**Reply by harikrishnan83** [reply]
@Kpizzle this issue has been fixed in [release 0.76.1](https://github.com/znsio/specmatic-node/releases/tag/0.76.1). Can you please check now after upgrading your npm installation?

**Reply by Kpizzle** [reply]
@harikrishnan83 Yes, this is now working! I can follow the documentation and have specmatic run the tests. Thanks very much!
